src/components/commentCount.tsx

```diff
@@ -5,15 +5,16 @@ import { css, SerializedStyles } from '@emotion/core';
 import { textSans } from '@guardian/src-foundations/typography';
 import { border } from '@guardian/src-foundations/palette';
 import { remSpace } from '@guardian/src-foundations';
-
+import { Option, map, withDefault } from 'types/option';
 import { Format } from 'format';
 import { getPillarStyles } from 'pillarStyles';
+import { pipe2 } from 'lib';
 
 
 // ----- Component ----- //
 
 interface Props extends Format {
-    count: number;
+    count: Option<number>;
     commentable: boolean;
```

Comment:
Will `commentable: false` always correspond to `count: None`? Are there situations where there may be comments even when `commentable` is set to `true`? Or situations where the count is `None` (rather than `0`), even when `commentable` is set to `true`? 🤔

I'm asking because we've now got two variables that represent whether to show a comment count, and perhaps `count` is all we need?

Reply:
That's a good point, we've used `commentable` in a few other places including whether to render discussion-rendering or not. I think it's also used to filter out articles in mapi.

It probably can be refactored though so that we just use one.